src/test/java/org/humancellatlas/ingest/core/service/MetadataUpdateServiceTest.java

```diff
@@ -40,21 +41,15 @@
     @MockBean
     private ValidationStateChangeService validationStateChangeService;
 
+    private ObjectMapper mapper = new ObjectMapper();
```

Comment:
Having a shared member can couse unexpected results if we decide to run tests in parallel.